Cleanup.ps1

```diff
@@ -0,0 +1,70 @@
+param([PSCredential]$Credential, [string]$TenantId)
+Import-Module AzureAD
+$ErrorActionPreference = 'Stop'
+
+
+Function Cleanup
+{
+<#
+.Description
+This function removes the Azure AD applications for the sample. These applications were created by the Configure.ps1 script
+#>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$True, HelpMessage='Tenant ID (This is a GUID which represents the "Directory ID" of the AzureAD tenant into which you want to create the apps')]
```

Comment:
This is a little confusing - is $tenantId mandatory? It does not seem like it. Personally I prefer to just throw instead of adding this.